pkg/apis/config/feature_flags.go

```diff
@@ -75,7 +77,13 @@ const (
 	enableAPIFields                     = "enable-api-fields"
 	sendCloudEventsForRuns              = "send-cloudevents-for-runs"
 	embeddedStatus                      = "embedded-status"
+	enableLoggingCredentialsFilter      = "enable-logging-credentials-filter"
 	enableSpire                         = "enable-spire"
+
+	// EnvEnableLoggingCredentialsFilter is the name of the environment variable that is set on the step containers
+	// when feature enable-logging-credentials-filter is enabled.
+	// #nosec G101 -- Its just the name of an environment variable that will carry credentials
+	EnvEnableLoggingCredentialsFilter = "TEKTON_FEATURE_ENABLE_LOGGING_CREDENTIALS_FILTER"
```

Comment:
It seems like this is just a boolean variable and possibly redundant with FeatureFlags.EnableLoggingCredentialsFilter-- can it be removed? Can we pass the boolean as a new optional arg to the entrypoint?

Reply:
Could the entrypoint just check to see if the downward API file is present, and if so, assume that credentials filtering is enabled?

Reply:
Actually that would mean you have a dynamic feature flag which could influence predictability and stability. And the feature flag is meant to ensure stability by allowing to simply turn something off. I like my feature flags simple and predictable.